### PR TITLE
Bugfixes/1848 issues with non english culture

### LINKED ISSUE #1848: Some tests fail if culture is different from en-US
**Describe the bug**
Running the tests in german culture yields 5 errors. These are all releated to differences in culture ("," instead of ".", localized error messages instead of the english ones).

**To Reproduce**
Run the tests with a German culture
- A) German WIndows
- B) Setting `System.Globalization.CultureInfo.CurrentCulture`/`System.Globalization.CultureInfo.CurrentUICulture` to German culture before any tests executes (untested)

**Expected behavior**
Tests should not vary because of culture of operating system.

**Information (please complete the following information):**
 - OS: Windows 11
 - Platform .NET 8
 - FluentMigrator version 5.2.0/source

### PR DESCRIPTION
**Pull request by warappa**
- Ensure tests checking against validation messages have invariant (english) culture
- Fix bug in `PostgresGenerator` if `IndexVacuumCleanupIndexScaleFactor` is used with a floating point number (should use invariant culture).

Fixes #1848 

> [!NOTE]
> Maybe other `ToString()` calls need to be investigated to ensure they do not cause the same issues as in `PostgresGenerator`.

### CODE REVIEW

File: test/FluentMigrator.Tests/Unit/Builders/Delete/DeleteDataExpressionTests.cs [hunk 1]
```diff
@@ -110,10 +111,13 @@ public void CallingIsNullAddsANullColumn()
         [Test]
         public void DefaultMigrationExpressionValidatorShouldReturnErrorWhenTableNameIsNotSpecified()
         {
+            // Ensure error messages are in English
+            CultureInfo.CurrentUICulture = CultureInfo.InvariantCulture;
```
Comment (jzabroski) [inline test/FluentMigrator.Tests/Unit/Builders/Delete/DeleteDataExpressionTests.cs:115]: Why not just use [Test, Culture("en-US")]

**Review by jzabroski — CHANGES_REQUESTED** [review]
Are you familiar with https://docs.nunit.org/articles/nunit/writing-tests/attributes/setculture.html ?  I have not used it but have used the xunit equivalent UseCulture.  In reading the docs, it seems extremely similar.  I am not sure if it will work side-by-side with Setup, but otherwise it should work great?  Note, nunit confusingly has a CultureAttribute, which is a _test filter_.  We want SetCultureAttribute, not CultureAttribute.  SetCultureAttribute sets the culture.  CultureAttribute filters it.

File: test/FluentMigrator.Tests/Unit/Builders/Delete/DeleteDataExpressionTests.cs (hunk 1 above)
Comment (warappa) [reply test/FluentMigrator.Tests/Unit/Builders/Delete/DeleteDataExpressionTests.cs:115]: I didn't know about this but this seems very handy! Thanks for pointing that out!👍

Comment (warappa) [reply test/FluentMigrator.Tests/Unit/Builders/Delete/DeleteDataExpressionTests.cs]: I use `SetUICultureAttribute`, as the validation messages rely on `CurrentUICulture` and not on `CurrentCulture` (tested).

Comment (jzabroski) [reply test/FluentMigrator.Tests/Unit/Builders/Delete/DeleteDataExpressionTests.cs]: thanks. will kick off ci service in a bit

Comment (warappa) [reply test/FluentMigrator.Tests/Unit/Builders/Delete/DeleteDataExpressionTests.cs]: Updated to empty string

**Pull request merged**